site/public/js/office-hours-queue.js

```diff
@@ -0,0 +1,61 @@
+//this helps update the frontend when the page refreshes because without this the sort icon would reset and the sort state would not
+document.addEventListener('DOMContentLoaded', () => {
+    const sortIndicator = document.getElementById('sort-indicator');
+    let sortState = localStorage.getItem('sort-indicator');
+
+    if (sortState === null) {
+        sortState = 'off';
+        localStorage.setItem('sort-indicator', sortState);
```

Comment:
This will set the key for the site globally as `sort-indicator`. Can you be more specific and add office hours to it or something?